frontend/src/app/processes/processes.component.ts

```diff
@@ -58,6 +63,10 @@ export class ProcessesComponent implements OnInit, OnDestroy {
     {item: 'redirect', viewValue: 'Redirect'},
     {item: 'embedded', viewValue: 'Embedded'}
   ];
+  graphTypes: GraphType[] = [
+    {item: 'Heuristic Net', viewValue: 'Heuristic Net'},
+    {item: 'DFG', viewValue: 'DFG'}
```

Comment:
```suggestion
    {item: 'HN', viewValue: 'Heuristic Net'},
    {item: 'DFG', viewValue: 'Directly-Follows Graph'}
```
I'm on mobile right now and can't check how this actually looks like but wouldn't it be nicer to tell the user what DFG actually means instead of using the acronym in the list?